### PR TITLE
feat: Add container spec for vsftpd and ps

### PR DESCRIPTION
**Pull request by wushiqinlou**
Signed-off-by: jiazhang <jiazhang@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

This is used to add container support for some prodsec rules.

### CODE REVIEW

**Comment by wushiqinlou** [comment]
@xiangce - It is ready for review, please check.

File: insights/specs/__init__.py [hunk 1]
```diff
@@ -787,4 +787,6 @@ class Specs(SpecSet):
     container_nginx_error_log = RegistryPoint(multi_output=True, filterable=True)
     container_installed_rpms = RegistryPoint(multi_output=True)
     container_inspect_keys = RegistryPoint(filterable=True)
+    container_vsftpd_conf = RegistryPoint(multi_output=True, filterable=True)
+    container_ps_aux = RegistryPoint(multi_output=True, filterable=True)
```
Comment (xiangce) [inline insights/specs/__init__.py:791]: nitpick, better re-sort these two items in alphabet order.

File: insights/specs/default.py [hunk 2]
```diff
@@ -691,4 +691,6 @@ class DefaultSpecs(Specs):
     container_nginx_conf = container_collect(container_nginx_conf_ds)
     container_nginx_error_log = container_collect(running_rhel_containers, "/var/log/nginx/error.log")
     container_redhat_release = container_collect(running_rhel_containers, "/etc/redhat-release")
+    container_vsftpd_conf = container_collect(running_rhel_containers, "/etc/vsftpd/vsftpd.conf")
+    container_ps_aux = container_execute(running_rhel_containers, "/bin/ps aux")
```
Comment (xiangce) [inline insights/specs/default.py:695]: The same comment as above.

**Review by xiangce — COMMENTED** [review]
Just a tiny suggestion, other than that, all look good to me.

**Review by xiangce — APPROVED** [review]

**Pull request merged**